cc/tiles/tile_manager.cc

```diff
@@ -162,6 +177,25 @@ class RasterTaskImpl : public TileTask {
   }
 
  private:
+#if defined(CASTANETS)
+  bool UsePartialMemorySync(gfx::Rect content_rect, gfx::Rect invalid_rect) {
```

Comment:
A trivial suggestion to make this function simpler.

```
  bool UsePartialMemorySync(gfx::Rect content_rect, gfx::Rect invalid_rect) {
    if (invalid_rect.IsEmpty() || content_rect == invalid_rect)
      return false;

    // If dirty area is greater than 50% of content area,
    // do not use partial sync.
    if (((float)invalid_rect.size().GetArea() / 
         (float)content_rect.size().GetArea()) > 0.5)
      return false;	  
    return true;
  }
```

Reply:
Thanks, done.